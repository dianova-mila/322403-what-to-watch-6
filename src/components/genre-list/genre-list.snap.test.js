import React from 'react';
import {render} from '@testing-library/react';
import GenreList from "./genre-list";

const genres = [`All genre`, `Drama`, `Crime`, `Thriller`];
const currentGenre = `All genre`;
const onSelectGenre = jest.fn();

it(`GenreList should render correctly`, () => {
  const {container} = render(<GenreList genres={genres} currentGenre={currentGenre} onSelectGenre={onSelectGenre}/>);

  expect(container).toMatchSnapshot();
});
