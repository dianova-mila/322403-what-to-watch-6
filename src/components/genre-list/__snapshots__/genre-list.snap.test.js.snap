// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList should render correctly 1`] = `
<div>
  <ul
    class="catalog__genres-list"
  >
    <li
      class="catalog__genres-item catalog__genres-item--active"
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        All genre
      </a>
    </li>
    <li
      class="catalog__genres-item "
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        Drama
      </a>
    </li>
    <li
      class="catalog__genres-item "
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        Crime
      </a>
    </li>
    <li
      class="catalog__genres-item "
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        Thriller
      </a>
    </li>
  </ul>
</div>
`;
